docs/static/releasenotes.asciidoc

```diff
@@ -16,6 +17,84 @@ This section summarizes the changes in the following releases:
+* Support for integration plugins in plugin manager https://github.com/elastic/logstash/pull/9811[#9811]
+* Promote Java execution from experimental to beta https://github.com/elastic/logstash/pull/10063[#10063]
+
+==== Plugins
+
+*Elastic App Search output*
+
+* New: Added as default plugin
+
+*SNMP input*
+
+* New: Added as default plugin
+
+*Elasticsearch filter*
+
+* Adds [@metadata][total_hits] with total hits returned from the query https://github.com/logstash-plugins/logstash-filter-elasticsearch/pull/106[#106]
+* Improves error logging to fully inspect caught exceptions https://github.com/logstash-plugins/logstash-filter-elasticsearch/pull/105[#105]
+
+*Translate filter*
+
+* Fix to align with docs - looked-up values are always strings. Coerce better.
+* Fix bug in dictionary/file the always applied RegexExact, manifested when dictionary keys are not regex compatible
+* Added info to dictionary_path description to explain why integers must be quoted
+* Fix bug in csv_file when LS config has CSV filter plugin specified as well as a csv dictionary.
+* Updated formatting of examples in documentation for consistent rendering
+* Add iterate_on setting to support fields that are arrays, see the docs for detailed explanation.
+* Add Rufus::Scheduler to provide asynchronous loading of dictionary.
+* Re-organise code, yields performance improvement of around 360%
+
+*Useragent filter*
+
+* Update source mapping to latest from uap-core https://github.com/logstash-plugins/logstash-filter-useragent/issues/53[#53]
+
+*Kafka input*
+
+* Upgrade Kafka client to version 2.0.0
+* Docs: Correct list formatting for decorate_events
+* Docs: Add kafka default to partition_assignment_strategy
+
+*Redis input*
+
+* Added support for renamed redis commands
+* Add channel to the event
+
+*S3 input*
+
+* Docs: Fixed link formatting for input type
+* Skips objects that are archived to AWS Glacier with a helpful log message (previously they would log as matched, but then fail to load events) https://github.com/logstash-plugins/logstash-input-s3/pull/160[#160]
+* Added watch_for_new_files option, enabling single-batch imports https://github.com/logstash-plugins/logstash-input-s3/pull/159[#159]
+
+*TCP input*
+
+* Added support for pkcs1 and pkcs8 key formats https://github.com/logstash-plugins/logstash-input-tcp/issues/122[#122]
+* Changed server-mode SSL to run on top of Netty https://github.com/logstash-plugins/logstash-input-tcp/issues/122[#122]
+* Changed travis testing infra to use logstash tarballs https://github.com/logstash-plugins/logstash-input-tcp/issues/122[#122]
+* Fixed certificate chain handling and validation https://github.com/logstash-plugins/logstash-input-tcp/issues/124[#124]
+* Added new configuration option dns_reverse_lookup_enabled to allow users to disable costly DNS reverse lookups https://github.com/logstash-plugins/logstash-input-tcp/issues/100[#100]
+
+*Netflow codec*
```

Comment:
Need a return after heading for list to render as a bulleted list. (As is, it renders as a paragraph.)